frontend/src/features/map/overlays/semaphores/SemaphoreCard.tsx

```diff
@@ -84,26 +85,46 @@ export function SemaphoreCard({ feature, selected = false }: { feature: any; sel
 
       <StyledContactContainer>
         {phoneNumber && (
-          <Whisper
-            controlId="phone-tooltip"
-            onClick={onCopyPhone}
-            placement="right"
-            speaker={hoverTooltip(tooltipPhoneState.text, tooltipPhoneState.className)}
-            trigger={tooltipPhoneState.trigger as OverlayTriggerType}
-          >
-            <StyledContact>Contact&nbsp;:&nbsp;{phoneNumber}</StyledContact>
-          </Whisper>
+          <StyledContactLine>
+            <Whisper
+              controlId="phone-tooltip"
+              onClick={onCopyPhone}
+              placement="right"
+              speaker={hoverTooltip(tooltipPhoneState.text, tooltipPhoneState.className)}
+              trigger={tooltipPhoneState.trigger as OverlayTriggerType}
+            >
+              <span>
+                <StyledCopyButton
+                  accent={Accent.TERTIARY}
+                  color={COLORS.slateGray}
+                  Icon={Icon.Duplicate}
+                  iconSize={20}
+                />
+              </span>
+            </Whisper>
+            <span>Contact&nbsp;:&nbsp;{phoneNumber}</span>
+          </StyledContactLine>
         )}
         {email && (
-          <Whisper
-            controlId="mail-tooltip"
-            onClick={onCopyMail}
-            placement="right"
-            speaker={hoverTooltip(tooltipMailState.text, tooltipMailState.className)}
-            trigger={tooltipMailState.trigger as OverlayTriggerType}
-          >
-            <StyledContact>{email}</StyledContact>
-          </Whisper>
+          <StyledContactLine>
+            <Whisper
+              controlId="mail-tooltip"
+              onClick={onCopyMail}
+              placement="right"
```

Comment:
Un détail que je viens de voir : en fait le tooltip se place sur le mail ou le num de tel, c'est peut être dommage ? 
```suggestion
              placement="left"
```